pkg/adapterManager/manager.go

```diff
@@ -56,15 +56,15 @@ var (
 	dispatchCounter = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: "mixer",
-			Subsystem: "adapter",
+			Subsystem: "adapter_old",
```

Comment:
`adapter` subsystem is now used by the "new adapter" code.
Therefore moving the older metric to "adapter_old" subsystem.
This will be removed once switch over is complete.